firefox-ios/Ecosia/Core/Environment/URLProvider.swift

```diff
@@ -196,6 +200,11 @@ public enum URLProvider {
         ["/accounts/sign-out"]
     }
 
+    /// URL paths that indicate profile page access
+    public var profilePaths: [String] {
+        ["/accounts/profile"]
+    }
```

Comment:
I wonder if we shouldn't use this inside `accountProfile`?

This actually is currently a bit repeated in other cases too like `signUpPaths` and `loginURL` or also  `signOutPaths` and `logoutURL`. Should we avoid repeating ourselves by having the path in one key variable which is used in other places?